test-it/karma/propertynode/PropertyNodeSequential.Test.js

```diff
@@ -0,0 +1,83 @@
+describe('The Vireo PropertyNode', function () {
+    'use strict';
+
+    var Vireo = window.NationalInstruments.Vireo.Vireo;
```

Comment:
How is uppercase Vireo different from lowercase vireo in this file? Can we just use one?

Reply:
This is just a pattern to make it easier to transition to ES6 modules which would use a `require()`. You can think of Vireo as the function constructor of a vireo object. We could, for example, only do a `vireo = new window.NationalInstruments.Vireo.Vireo()` in the  `beforeEach()` and remove this line.

Reply:
Also the uppercase is a reference to the class itself while the lowercase is a reference to an instance of the class inside this file. One of our linting rules enforces that classes start with uppercase names.